client/src/components/HistoryViewer/HistoryViewerVersionList.js

```diff
@@ -9,20 +9,27 @@ import { messageType } from 'types/messageType';
 import { versionType } from 'types/versionType';
 import { compareType } from 'types/compareType';
 
-class HistoryViewerVersionList extends PureComponent {
+const HistoryViewerVersionList = ({
```

Comment:
Arguably to convert from a `PureComponent` we should wrap the whole thing in `memo`, e.g

```js
const MyComponent = memo(({ myProp }) => {
    // ...
    return "some rendered stuff here";
});
```

Probably not worth doing though and I'm not 100% confident that it would be without problems.